test/integration/parachain/staging/sequential/oracle.test.ts

```diff
@@ -6,47 +6,53 @@ import Big from "big.js";
-    let collateralCurrency: Currency<CollateralUnit>;
+    let collateralCurrencies: Array<Currency<CollateralUnit>>;
     let oracleAccount: KeyringPair;
 
     before(async () => {
         api = await createSubstrateAPI(PARACHAIN_ENDPOINT);
         const keyring = new Keyring({ type: "sr25519" });
         oracleAccount = keyring.addFromUri(ORACLE_URI);
         interBtcAPI = new DefaultInterBtcApi(api, "regtest", oracleAccount, ESPLORA_BASE_PATH);
-        collateralCurrency = getCorrespondingCollateralCurrency(interBtcAPI.getGovernanceCurrency()) as Currency<CollateralUnit>;
-        wrappedCurrency = interBtcAPI.getWrappedCurrency();
+        collateralCurrencies = getCorrespondingCollateralCurrencies(interBtcAPI.getGovernanceCurrency()) as Array<Currency<CollateralUnit>>;
     });
 
     after(() => {
         return api.disconnect();
     });
 
     it("should set exchange rate", async () => {
-        const exchangeRateValue = new Big("3913.7424920372646687827621");
-        const newExchangeRate = new ExchangeRate<
-            Bitcoin,
-            BitcoinUnit,
-            typeof collateralCurrency,
-            typeof collateralCurrency.units
-        >(Bitcoin, collateralCurrency, exchangeRateValue);
-        await interBtcAPI.oracle.setExchangeRate(newExchangeRate);
-        await interBtcAPI.oracle.waitForExchangeRateUpdate(newExchangeRate);
+        for (const collateralCurrency of collateralCurrencies) {
+            const exchangeRateValue = new Big("3913.7424920372646687827621");
+            const newExchangeRate = new ExchangeRate<
+                Bitcoin,
+                BitcoinUnit,
+                typeof collateralCurrency,
+                typeof collateralCurrency.units
+            >(Bitcoin, collateralCurrency, exchangeRateValue);
+            await interBtcAPI.oracle.setExchangeRate(newExchangeRate);
+            await interBtcAPI.oracle.waitForExchangeRateUpdate(newExchangeRate);
+        }
     });
 
-    it("should convert satoshi to planck", async () => {
-        const bitcoinAmount = BitcoinAmount.from.BTC(100);
-        const exchangeRate = await interBtcAPI.oracle.getExchangeRate(collateralCurrency);
-        const expectedCollateral = exchangeRate.toBig(undefined).mul(bitcoinAmount.toBig(BitcoinUnit.BTC)).round(0, 0);
-
-        const collateralAmount = await interBtcAPI.oracle.convertWrappedToCurrency(bitcoinAmount, collateralCurrency);
-        assert.equal(collateralAmount.toBig(collateralCurrency.base).round(0, 0).toString(), expectedCollateral.toString());
+    it("should convert satoshi to collateral currency", async () => {
+        for(const collateralCurrency of collateralCurrencies) {
+            const bitcoinAmount = BitcoinAmount.from.BTC(100);
+            const exchangeRate = await interBtcAPI.oracle.getExchangeRate(collateralCurrency);
+            const expectedCollateral = exchangeRate.toBig(undefined).mul(bitcoinAmount.toBig(BitcoinUnit.BTC)).round(0, 0);
```

Comment:
Why `undefined`?

Reply:
Good catch! That was supposed to be `exchangeRate.toBig()` - looks like me stripping the previously defined, single collateralCurrency, auto-refactored that into `undefined`.
Awkward, but at least not breaking.